### PR TITLE
Suggest the correct markdownlint CLI

### PR DESCRIPTION
**Pull request by rakyll**
So the developers can run `make markdownlint` to lint the files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,9 +95,9 @@ the box settings for this repository will be consistent.
 To check for style violations, use
 
 ```bash
-# Ruby and gem are required for mdl
-gem install mdl
-mdl -c .mdlrc .
+# npm is required to install markdownlint
+npm install -g markdownlint-cli
```
Comment (yurishkuro) [inline CONTRIBUTING.md:99]: I would suggest creating a package.json and installing locally instead of relying on global install.

Comment (rakyll) [reply CONTRIBUTING.md:99]: Developers are advanced enough to fallback to that option if they don't want to install global. This enables `make markdownlint` on a development machine which was the actual purpose of the change.

Comment (anuraaga) [reply CONTRIBUTING.md:99]: I don't think it has to block this change, but adding a docker image for markdownlint to the build-tools would be nice

Comment (jsuereth) [reply CONTRIBUTING.md:99]: +1 to the docker image idea.   Also, hope `make check` and `make fix` helps others :)

Does this tool have an auto-fix option?

**Review by jsuereth — APPROVED** [review]

**Comment by rakyll** [comment]
Please feel free to follow up with a docker image and fix options.

**Review by reyang — APPROVED** [review]
LGTM.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,9 +95,9 @@ the box settings for this repository will be consistent.
 To check for style violations, use
 
 ```bash
-# Ruby and gem are required for mdl
-gem install mdl
-mdl -c .mdlrc .
+# npm is required to install markdownlint
```
Comment (SergeyKanzhelev) [inline CONTRIBUTING.md:98]: nit

```suggestion
# npm is required to install markdownlint-cli
```

**Review by SergeyKanzhelev — APPROVED** [review]

**Review by anuraaga — APPROVED** [review]

**Review by arminru — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (bogdandrutu) [reply CONTRIBUTING.md:98]: We already have a make rule for this, what about `make install-markdownlint` or just `make install-tools`

**Comment by rakyll** [comment]
PTAL, switched to `make install-markdownlint`.

**Review by arminru — APPROVED** [review]

**Comment by jmacd** [comment]
(I was assigned this, and I was looking at the unresolved comment yesterday -- why it wasn't merged.)

**Pull request merged**